src/components/DepositWidget/Row.tsx

```diff
@@ -116,13 +118,14 @@ export const Row: React.FC<RowProps> = (props: RowProps) => {
             <>{withdrawing && spinner}0</>
           )}
         </td>
-        <td data-label="Wallet" title={formatAmountFull({ amount: walletBalance, precision: decimals }) || ''}>
+        <td data-label="Wallet">
           {isWeth ? (
             <ul>
-              <li className="not-implemented">
-                0.1 ETH <WrapEtherBtn label="Wrap" className="wrapUnwrapEther" />
+              <li title={ethBalance ? formatAmountFull({ amount: ethBalance, precision: decimals }) : undefined}>
```

Comment:
here `title={ethBalance ? ... : undefined}`
and below it's `title={... || ''}`
pick one, `'' or undefined`